megamek/src/megamek/common/Entity.java

```diff
@@ -10046,6 +10046,9 @@ public boolean isEligibleForPhysical() {
                 && hasWorkingMisc(MiscType.F_TOOLS,
                         MiscType.S_DEMOLITION_CHARGE)) {
             Hex hex = game.getBoard().getHex(getPosition());
+            if (hex == null) {
```

Comment:
there is a check not on board later in this function, should that be moved before this block.  to protect other calls beside just this one also?
```
       if (getPosition() == null) {
            return false; // not on board?
        }
```

Reply:
Good find!  I've moved the getPosition() check to the top.  I will keep the hex == null check as well, as I have a sneaking suspicion it's possible for units with a  non-null position to still have a null hex value...